markdown/articles/2021/02/10/alignment.md

```diff
@@ -0,0 +1,330 @@
+---
+authors: Doishy
+banner: Ideas unbound
```

Comment:
See https://github.com/angrybacon/doomsday-wiki#banners to retrieve the art :nerd_face: